birankpy/README.md

```diff
@@ -0,0 +1,184 @@
+# API documentation of birankpy
+
+## pagerank
+```python
+pagerank(adj, d=0.85, max_iter=200, tol=0.0001, verbose=False)
+```
+
+Return the PageRank of the nodes in a graph.
+This funcion is replica of networkx's pagerank_scipy method with modification.
```

Comment:
what's the license of networkx?

Reply:
Good point. Let me try something else

Reply:
they may allow modification/redistribution of the source code